package.json

```diff
@@ -44,33 +44,30 @@
         "build": "rollup -c",
         "dev": "karma start ./karma.conf.cjs --auto-watch --no-single-run --browsers chrome --grep",
         "dev:ff": "karma start ./karma.conf.cjs --auto-watch --no-single-run --browsers firefox --grep",
-        "docs": "npm run build && vuepress build docs --no-cache",
-        "docs:dev": "npm run build && vuepress dev docs --no-cache",
+        "docs": "pnpm build && pnpm --filter docs build",
+        "docs:dev": "pnpm build && pnpm --filter docs dev",
```

Comment:
Extending on my previous comment, does this still work since vuepress is not in here anymore but in the child package.json

Reply:
yes, this works. the `--filter docs` means to run the given command on the `docs` project. happy to write it as `cd docs && pnpm dev` if that's clearer

Reply:
No think this is fine, should have read the docs, but kind of read over the filter, bit tired so in my head it was the no cache part for some reason.